examples/inherit/single/one/suite.rc

```diff
@@ -11,8 +11,8 @@ title = "User Guide [runtime] example."
                  OBS:succeed-all => bar"""
 [runtime]
     [[root]] # base namespace for all tasks (defines suite-wide defaults)
-        [[[job submission]]]
-            method = at_now
```

Comment:
This was a very old example before `at_now` was renamed to `at` (so in theory we can run `at` with different arguments, now just `now`).